catfs/fs.go

```diff
@@ -1006,7 +1006,7 @@ func (fs *FS) Stage(path string, r io.Reader) error {
 		hint = fs.hintFetcher.Lookup(path)
 	}
 
-	stream, err := mio.NewInStream(sizeReader, path, key, hint)
+	stream, isRaw, err := mio.NewInStream(sizeReader, path, key, hint)
```

Comment:
Would it be better to have the `isRaw` field in the stream structure? Some of our stream already have extra info and it is convenient to keep it in one place.

Reply:
I made it a return parameter because it's something that needs to be stored persistently outside of the stream (in this case as part of the metadata). Making it a part of stream would suggest otherwise. Another downside is that stream often is passed through as ``io.Reader`` or ``io.ReadSeeker``. This would strip access to the ``isRaw`` attribute from the stream.